src/leetcode/NumberHelper.java

```diff
@@ -0,0 +1,17 @@
+package leetcode;
+
+import java.util.Set;
+import java.util.TreeSet;
+
+public class NumberHelper {
+
+    public static boolean containsDuplicate(int[] numbers) {
+        Set<Integer> sortedNumbers = new TreeSet<>();
```

Comment:
the tree is by default sorted and Integer is a number, the name needs to be improved